zerver/openapi/zulip.yaml

```diff
@@ -15718,6 +15733,10 @@ paths:
                               **Changes**: In Zulip 8.0 (feature level 227), added `DMs, mentions,
                               and followed topics` option, renumbering the options to insert it in
                               order.
+                          mobile_push_notifications_enabled:
+                            type: boolean
+                            description: |
+                              Enable mobile push notifications for direct messages and @-mentions.
```

Comment:
I don't get why this is added. This PR is about default push notification setting for streams right?